tests/lib/rules/line-comment-position.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * @fileoverview enforce position of line comments
+ * @author Alberto Rodríguez
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/line-comment-position"),
+
```

Comment:
Unnecessary line break? (I know this comes from the Yeoman generator, hope to fix that someday)